tests/unit/lms/validation/_api_test.py

```diff
@@ -144,3 +134,18 @@ def all_fields(self):
             "lis_result_sourcedid": "modelstudent-assignment1",
             "score": 0.5,
         }
+
+
+@pytest.fixture
+def json_request(pyramid_request):
+    def _make_json_request(data, exclude=None):
+        pyramid_request.content_type = "application/json"
+
+        if exclude:
+            for field in exclude:
+                data.pop(field, None)
+
+        pyramid_request.body = json.dumps(data)
+        return pyramid_request
```

Comment:
I think this function returns the same object instance each time you call it

Reply:
(The fix would be as suggested above)

Reply:
Right, so if any single test called it twice, it would be mutating the same instance. We just happen to be ok here because there's only ever one request.

I'm going to assume this is ok for now.